controllers/workspace/finalize.go

```diff
@@ -45,10 +45,12 @@ const (
 )
 
 var (
-	cleanupJobCompletions  = int32(1)
-	cleanupJobBackoffLimit = int32(3)
-	// PVCCleanupPodMemoryLimit is the memory limit used for PVC clean up pods
-	PVCCleanupPodMemoryLimit = resource.MustParse("32Mi")
+	cleanupJobCompletions      = int32(1)
+	cleanupJobBackoffLimit     = int32(3)
+	PVCCleanupPodMemoryLimit   = resource.MustParse("32Mi")
+	PVCCleanupPodMemoryRequest = PVCCleanupPodMemoryLimit
+	PVCCleanupPodCPULimit      = resource.MustParse("10m")
```

Comment:
Maybe we should use `50m` or something, just in case? I think `10m` suggests 0.01 of a CPU core

Reply:
Limit is changed to 50m (0.05 CPU Core), request is still 5m(0.005 CPU Core), should it be fine?